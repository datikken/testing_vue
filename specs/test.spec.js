import TestComponent from '@/test';
import { mount, shallowMount } from '@vue/test-utils';
import List from "../src/list";

test('mount vue component', () => {
    const wrap = mount(TestComponent, {
      propsData: {
          value: 'vue-yes'
      }
    })

    expect(wrap.html()).toMatchSnapshot()
});

test(`List component shallow`, () => {
    console.log(mount(List).html())
    console.log(shallowMount(List).html())
});