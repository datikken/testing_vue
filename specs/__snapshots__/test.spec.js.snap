// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount vue component 1`] = `
<div>
  <h1>Hello vue-yes</h1>
</div>
`;
